tests/acceptance/posts-test.js

```diff
@@ -73,15 +73,15 @@ module('Acceptance: title', function(hooks) {
     await visit('/feeds/tomster');
     assert.equal(title(), 'Tomster (@tomster)');
 
-    await click('#zoey');
+    click('#zoey');
     await waitUntil(() => {
-      return find('div').innerText !== 'Loading...';
+      return !find('div[data-test-substate-loading]');
```

Comment:
This find probably needs to be imported

Reply:
find helper is imported, I'll use === null expression instead to give better intent what is meant with waiting element to disappear.

Reply:
Doh! I misread that as the TypeScript syntax 🤦‍♂️.